docs/data/material/components/menus/NestedMenu.tsx

```diff
@@ -0,0 +1,287 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import {
+  Box,
+  ClickAwayListener,
+  Grow,
+  Icon,
+  MenuList,
+  Paper,
+  Popper,
+  Typography,
+} from '@mui/material';
+
+export default function BasicMenu() {
```

Comment:
The exported function name should be the same as the file name.
There is a linter rule for this on master. Make sure to merge in the latest one.

Reply:
done